app/src/main/java/com/nononsenseapps/feeder/model/opml/OPMLImporter.kt

```diff
@@ -25,12 +27,22 @@ open class OPMLImporter(override val di: DI) : OPMLParserHandler, DIAware {
 
     override suspend fun saveFeed(feed: Feed) {
         val existing = feedDao.loadFeedWithUrl(feed.url)
-
+        val existingNostrFeeds = feedDao.getAllFeeds().filter { it.url.isAdaptedUrlFromNostrUri() }
         // Don't want to remove existing feed on OPML imports
         if (existing != null) {
             feedDao.updateFeed(feed.copy(id = existing.id))
         } else {
-            feedDao.insertFeed(feed)
+            if (feed.url.isAdaptedUrlFromNostrUri()) {
+                val potentialNostrFeed = existingNostrFeeds.find { it.title == feed.title }
```

Comment:
I don't think this is a very good idea. Feed titles are not unique.

If a user wants to add a second copy of a feed with a different url, then let them.

Reply:
OK, will fix.

Reply:
Fixed.